ansible/roles/package-upgrade/tasks/partials/apt.yml

```diff
@@ -4,6 +4,9 @@
 # Updates all packages for debian-based distributions
 #
 
+- name: update repositories
+  raw: apt-get update -y
```

Comment:
I think `update_cache=yes` does the same below, so this command seems to be unnecessary.

Reply:
Does this mean the whole `debian7.yml` is unnecessary ?

Reply:
@al-k21 It does seem like this can be removed, unless there's a reason to rerun it later in the setup process.

cc/ @jbergstroem in case there is a reason for it.